### DISCUSSION TITLE
2 Wemos D1 Mini mit Rs485 abgeraucht

### DISCUSSION THREAD

**Discussion by Padrino1974** [softwarecrash/EPEver2MQTT#50] [General]
Moin erst einmal danke für die Super Anleitung und Deine Arbeit. Ich habe folgendes Problem. 

Ich habe 2 EPEVER 1x2206 und 1x4210, gemäß Deiner Anleitung D1 Mini und Max xx Rs485 Adapter angeschlossen. Hat beides auf Anhieb funktioniert. EP`s sind im Web Frontend und sauber im HA mit MQTT. Strom kommt über 5 V D1mini auf den RS485, nach ca. 3 h beide Minis tot. Sind seit dem defekt der TTL leuchtet aber noch rot. Seit dem komme ich auch mit komplett neuen Komponenten nicht mehr auf den EP. Ist Dir bekannt ob  es eine Umstellung gab, die Geräte sind vor 14 Tagen beschafft worden.  Das Bedienen der Load Taste auf dem EP Ever führt zu einem kur veränderten blinken. Am Web Frontend wird jedoch nichts angezeigt... Bedienung wie Update und Neustart funktionieren aber komischerweise!?

**Comment by Padrino1974** [comment]
Ach der Baustein auf dem D1mini der ohne Funktion nach dem Verbinden extrem heiß wird ist der 4A2D

**Comment by all-solutions** [comment]
Mess mal bitte die Spannung, die aus dem Epever kommt.
Wenn der Wemos heiß wird, wäre das ein Anzeichen, dass da mehr als 5V rauskommen. 
Ansonsten mach mal bitte ein paar Fotos der kompletten Verkabelung.

**Reply by Padrino1974** [reply]
Moin aus dem IP Ever 2240 kommen 3,578 V raus, und auf dem 2206 3,543 habe bis jetzt 2 alt Wemos und Rs485 Platinen verwendet, bekomme nirgends mehr Daten ausgelesen... hatte bis jetzt noch nie EPS Probleme meine Growatts 3000 und 1500 Daly laufen auch über die von mir gebauten Platinen!

**Reply by Padrino1974** [reply]
okay hatte ich seinerzeit nicht fotografiert das einige Foto das ich noch habe ist folgendes

![IMG-20240825-WA0011](https://github.com/user-attachments/assets/b5cbbb29-cd31-4fbc-a6c4-24fa98fd681e)

**Comment by Padrino1974** [comment]
![IMG-20240825-WA0012](https://github.com/user-attachments/assets/1b0ac7d7-3a80-49ea-b0cc-16b66b64cda8)
![Uploading IMG-202
![IMG-20240825-WA0007](https://github.com/user-attachments/assets/09303b99-c669-4a99-9c71-4aca1c5ef286)
40825-WA0019.jpeg…]()
![20240823_141916](https://github.com/user-attachments/assets/9f6d373e-abe7-4b74-b56c-19d4c0335f8e)

**Comment by Padrino1974** [comment]
![Uploading IMG-20240825-WA0007.jpeg…]()

**Comment by Padrino1974** [comment]
![IMG-20240825-WA0019](https://github.com/user-attachments/assets/84830861-cd3c-4bb7-818e-ba25ee5292b6)

**Comment by Padrino1974** [comment]
![IMG-20240825-WA0007](https://github.com/user-attachments/assets/a7755786-d415-4ec9-8a87-4745e8a46056)
